acceptance-tests/tests/src/test/resources/jsonrpc/engine/prague/genesis.json

```
@@ -14,7 +14,7 @@
     "londonBlock":0,
     "terminalTotalDifficulty":0,
     "cancunTime":0,
-    "experimentalEipsTime":20,
+    "pragueTime":0,
```

```suggestion
    "pragueTime":20,
```

I think this might still want to be 20 because the point of these tests is to test the transition from cancun -> prague.
Hopefully that'll fix your AT failures, but that might be wishful thinking :)

---

There were a couple of other things I had to fix, but yeah, this was the one that made me spend an hour braking my brain!